### DISCUSSION TITLE
llidael plugin host with VSTi

### DISCUSSION THREAD

**Discussion by subderground** [DISTRHO/Cardinal#621] [Q&A] [answered]
Hi,
I'm trying the llidael plugin host with VSTi. For me it seems like it's only made for audio fx plug-ins (2 ins / 2 outs)... but not sure at all :-)
Have I missed something to send cv-notes and gates to the plug-in in the llidael plugin host ? A tricks ?

If not possible...it would be awesome ;-)
Have a great day.
T.

**Comment by falkTX** [comment, marked as answer]
you need to add an expander module to allow midi in and out. those expanders work for both carla and ildaeil

**Reply by subderground** [reply]
Hello,
Thank you. It works now with vst2 instruments, but still no sound with VST3 instruments (no problem with VST3fx).

The "show custom GUI doesn't work too (instruments or fx).
Must I install Carla too? (https://kx.studio/News/?action=view&url=carla-256-released)

Many thanks.
Toph  ( Windows 11 pc)

**Reply by falkTX** [reply]
no, only cardinal is needed. the vst3 host support is kinda new so make sure your cardinal version is up to date.
or try one of the nightly builds from https://github.com/DISTRHO/Cardinal/actions/workflows/build.yml

**Reply by subderground** [reply]
I allways run the last updated version. 😀
